dot-atom/styles.less

```diff
@@ -4,22 +4,21 @@
  * This stylesheet is loaded when Atom starts up and is reloaded automatically
  * when it is changed.
  *
- * If you are unfamiliar with LESS, you can read more about it here:
- * http://www.lesscss.org
```

Comment:
I kind of liked the link here which helps users learn more about less. If you don't feel strongly about removing this, could I talk you into keeping the link or adding some other links to point users to :books:?